.cspell.json

```diff
@@ -1139,6 +1139,7 @@
     "odometry",
     "Ofast",
     "Offroad",
+    "offsetted",
```

Comment:
I think it should be replaced by `offset`.
https://en.wiktionary.org/wiki/offset#English

> offset (third-person singular simple present [offsets](https://en.wiktionary.org/wiki/offsets#English), present participle [offsetting](https://en.wiktionary.org/wiki/offsetting#English), simple past and past participle offset or (rare) [offsetted](https://en.wiktionary.org/wiki/offsetted#English))

Reply:
If you don't think so, please show some concrete examples or references.

Reply:
When adding the word `offset`, the word `offsetted` can be ignored as well?

Reply:
No. You need to replace the word used in your code.
However, if there are any rationales, adding this word is okay.

Reply:
Then I wanna add `offsetted`.

First of all, I guess `offset` is a popular English word, so we do not have to register.
But cspell does not know the past tense of `offset` which is `offsetted`.
https://ejje.weblio.jp/content/offsetted

Reply:
@takayuki5168 As I wrote in https://github.com/tier4/autoware-spell-check-dict/pull/477#discussion_r1109160630, the most popular past tense of `offset` is `offset`.
So, I believe you should follow the major usage.

Reply:
Ah, now I got what you meant.
Okay, then we can replace offsetted with offset.